packages/config/__snapshots__/index_spec.ts.js

```diff
@@ -0,0 +1,153 @@
+exports['src/index .getBreakingKeys returns list of breaking config keys 1'] = [
```

Comment:
I think we need to delete the existing snapshots: https://github.com/cypress-io/cypress/tree/develop/packages/config/__snapshots__